src/sofa/pbrpc/rpc_server_impl.cc

```diff
@@ -70,6 +74,7 @@ RpcServerImpl::~RpcServerImpl()
     SOFA_PBRPC_FUNCTION_TRACE;
     Stop();
     _service_pool.reset();
+    _web_service.reset();
```

Comment:
_web_service.reset() should be put before _service_pool.reset(). first create, last destroy.